.github/CONTRIBUTING.md

```diff
@@ -305,81 +305,30 @@ To setup Development Environment without Docker you need following pre-requisite
 -   postgres >= 12
 -   dbt 1.4.x or 1.5.x
 
-eg. on MacOS you can follow this instructions:
+> [!TIP] Mac Installs
+> On MacOS you can run an [install script](scripts/local-mac.sh) to install all the dependencies and setup the development environment.
 
 ```shell
-# 1 Install Homebrew (https://brew.sh)
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-
-# 2 Install nvm (https://github.com/nvm-sh/nvm#troubleshooting-on-macos) and other required dependencies
-brew update
-brew install nvm
-brew install pkg-config cairo pango libpng jpeg giflib librsvg pixman python-setuptools
-
-# 3 Install specified node version using NVM (https://github.com/nvm-sh/nvm)
-
-nvm install v20.8.0
-nvm alias default v20.8.0
-
-# 4 Install postgres (https://wiki.postgresql.org/wiki/Homebrew) and pgvector
-brew install postgresql@14
-brew services start postgresql@14
-
-# pgvector is an extension for postgres we use in Lightdash, it needs to be installed separately
-# More info about this extension and a detailed installation guide available here: https://github.com/pgvector/pgvector
-# on Linux, you can install `postgresql-14-pgvector`, available on apt
-# You might need to point pgvector to a correct postgres instance if you have multiple versions installed
-# export PG_CONFIG=/opt/homebrew/opt/postgresql@14/bin/pg_config
-git clone --branch v0.8.0 https://github.com/pgvector/pgvector.git && cd pgvector && make && sudo make install && cd ..
-
-# 5 Install dbt using pip
-# Detailed installation guide available here: https://docs.getdbt.com/docs/core/pip-install
-# Create python virtual env
-python3 -m venv env-lightdash # or your preferred env name
-# Activate the env
-# You can deactivate python virtual env by running `deactivate` later
-source env-lightdash/bin/activate
-
-python -m pip install dbt-postgres==1.4.9
-
-# 6 Clone the repo and open it in your IDE
-git clone https://github.com/lightdash/lightdash.git
-cd lightdash
-
-# 7 Copy `.env.development` to `.env.development.local`
-cp .env.development .env.development.local
-
-# 8 Edit some environment variables to match your setup
-open .env.development.local -t
-
-# 8.1 You may need to edit the following variables:
-PGHOST=localhost
-PGPORT=5432
-PGUSER=pg_user *OR* machine username if no prior postgres set up
-PGPASSWORD=pg_password *OR* blank if no prior postgres set up
-PGDATABASE=postgres
-DBT_DEMO_DIR=$PWD/examples/full-jaffle-shop-demo
-
-# 9 Install packages
-pnpm install
+# Run the install script
+./scripts/local-mac.sh

```

Comment:
@stevelikesmusic did you test this script? Is it idempotent? What happens if I have some dependencies already? the pro of the instructions is that it shows each step and devs can tweak as needed for their machine.